code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -341,6 +349,7 @@
 	metabolization_rate = 0.1
 	penetrates_skin = 1
 	can_synth = 0
+	taste_message = null
```

Comment:
Rads don't taste and I'd rather keep all the toxins rather ambiguous, so as not to give people the ability to check the code for what they drank.